examples/pr_2D_FFT_line_sensor/pr_2D_FFT_line_sensor.py

```diff
@@ -77,11 +75,10 @@ def main():
 
     # run the simulation
     sensor_data = kspaceFirstOrder2D(kgrid, source, sensor, medium, simulation_options, execution_options)
-    sensor_data = sensor_data['p'].T
+    sensor_data = sensor_data["p"].T
 
     # reconstruct the initial pressure
-    p_xy = kspaceLineRecon(sensor_data.T, dy=d[1], dt=kgrid.dt.item(), c=medium.sound_speed.item(),
-                           pos_cond=True, interp='linear')
+    p_xy = kspaceLineRecon(sensor_data.T, dy=d[1], dt=kgrid.dt.item(), c=medium.sound_speed.item(), pos_cond=True, interp="linear")
```

Comment:
Can we have step which converts arguments to named arguments? I find named arguments useful for avoiding bugs, wdyt?

Reply:
Do you know which rule we would have to use? I could only find [this open issue](https://github.com/astral-sh/ruff/issues/8382).